applications/RANSApplication/custom_utilities/rans_calculation_utilities.cpp

```diff
@@ -82,6 +112,25 @@ array_1d<double, 3> EvaluateInPoint(
     return value;
 }
 
+template <typename TDataType>
+TDataType EvaluateInParentCenter(
+    const Variable<TDataType>& rVariable,
+    const ConditionType& rCondition,
+    const int Step)
+{
+    const ElementType& r_parent_element = rCondition.GetValue(NEIGHBOUR_ELEMENTS)[0];
+    const GeometryType& r_parent_geometry = r_parent_element.GetGeometry();
+
+    Vector parent_gauss_weights;
+    Matrix parent_shape_functions;
+    GeometryData::ShapeFunctionsGradientsType parent_shape_function_derivatives;
+    CalculateGeometryData(r_parent_geometry, GeometryData::IntegrationMethod::GI_GAUSS_1,
+                          parent_gauss_weights, parent_shape_functions,
+                          parent_shape_function_derivatives);
+    return EvaluateInPoint(r_parent_geometry, rVariable,
+                           row(parent_shape_functions, 0), Step);
```

Comment:
Wouldn't it be more efficient to get the parent nodal values and directly average them?

Reply:
removed :)